src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -511,7 +511,7 @@ public static bool IsUnsafe([NotNullWhen(returnValue: true)] this ISymbol? membe
         }
 
         public static ITypeSymbol ConvertToType(
```

Comment:
Is Compilation.ObjectType guaranteed to be non-null?

Reply:
I'm not sure. If not it will be updated when `Compilation` is annotated.